### ISSUE TITLE
Loading Contacts and Reconnecting

### ISSUE THREAD

**Issue by klonamy** [yakyak/yakyak#557]
ref: https://github.com/yakyak/yakyak/issues/111
I have the same trouble.

On the first login it is stuck at loading contacts.
On second start it is stuck at Connecting or says it can't connect and switching with connecting.

**OS:**
Windows 10 (64bit)

**Program-Version:**
yakyak 1.4.0 x64
yakyak 1.4.1 x64
yakyak 1.4.2 x64 _and_ ia32

So ist there a way to get a debug log?

**Comment by davibe** [comment]
You can open the window inspector with alt+ctrl+i if the main window has been loaded

You can also launch yakyak from cmd (windows 10 command line) by dragging `yakyak.exe` in the `cmd` window and then pressing enter. This will give you some debug output in the cmd window.

Have you tried to remove the appdata yakyak folder ? 
i.e.

<img width="1346" alt="screenshot 2017-01-16 07 54 42" src="https://cloud.githubusercontent.com/assets/123929/21973456/19d8838c-dbc1-11e6-991b-1cb54384effe.png">

**Comment by ockle** [comment]
I have the same problem on macOS. Using 1.4.2. Can't get past "Loading contacts". Google do email me about a new sign in. The only thing that ever shows in the network monitor are two fonts being loaded. Only thing in console is the version message.

**Comment by klonamy** [comment]
Okay I think i have the issue. I have 2fa active. (https://github.com/yakyak/yakyak/issues/402)
And btw. the deletion of the appdata folder of YakYak i had done many times.
I will close this for now.

**Comment by SapphireTactician** [comment]
I'm not using 2FA and I'm still stuck Loading Contacts. I'm using the Linux x86-64 binary with version 1.4.1. If I use the pre-release 1.4.2 then it's stuck on Connecting. Regardless, I haven't had a successful login.

**Comment by averissimo** [comment]
@SapphireTactician can you try starting with the console `./yakyak` and screenshot it?

Also open the inspector `ctrl + alt + i` and screenshot it

thanks

**Comment by SapphireTactician** [comment]
@averissimo My apologies for the late reply. These were taken with the pre-built binary of 1.4.1. Two screenshots of the terminal output, and the inspector output as instructed.

![2017-02-19--144421_1600x900](https://cloud.githubusercontent.com/assets/24427487/23105790/c3bc17ae-f6b2-11e6-8cd9-89663a04c044.png)
![2017-02-19--144434_1600x900](https://cloud.githubusercontent.com/assets/24427487/23105791/c3bcc898-f6b2-11e6-878c-5fccde0d9a2d.png)
![2017-02-19--144518_1600x900](https://cloud.githubusercontent.com/assets/24427487/23105792/c3c03f78-f6b2-11e6-8d3f-1b2f2cdccf90.png)

**Comment by averissimo** [comment]
try to replace the first two screenshots and censor the link that starts with `https://accounts...` (just to be safe)

I'll try to create a more verbose version of yakyak to see if we can find out why the screen does not change (as you are connected)

**Comment by averissimo** [comment]
but before doing that can you try to remove the yakyak folder as @davibe suggested? `~/.config/yakyak`

**Comment by SapphireTactician** [comment]
I've purged the config folder multiple times on my own when trying to troubleshoot it. No dice :(

**Comment by kirill133** [comment]
Similar behavior in my case - stack on "Connecting..." but when I've added one contact - connected successfully. So maybe it just hangs when new google account is used without any hangouts chats

**Comment by averissimo** [comment]
@kirill133 humm seems plausible and I think someone reported before, although I cannot pinpoint the issue.

Can you create a new issue with this particular bug (new account without any contacts)

**Comment by averissimo** [comment]
@SapphireTactician sorry for not replying but I've been off "yakyak" since my last comment, I'm going to think this through to see how we're going to solve this problem :)

**Comment by SapphireTactician** [comment]
@kirill133 I originally had no contacts, but then figured that might be the issue as well. I added a gmail user to my address book via Google Contacts but no change. I then logged into Hangouts via my browser, purged everything from yakyak and tried once more, no change.

**Comment by kirill133** [comment]
@SapphireTactician Yes, sorry, I was using the wrong word - it actually started to show main window only after adding one conversation (from my another gmail account), not contact.

**Comment by SapphireTactician** [comment]
@kirill133 You found the silver bullet! @averissimo After messaging my GF via the Hangouts website, I was able to launch YakYak successfully. Looks like @kirill133 isolated the bug for us. :1st_place_medal:

**Comment by averissimo** [comment]
@kirill133 this is a strange bug, thanks for workaround!!
